.github/workflows/ci.yml

```
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - 'lts'
```

```suggestion
          - 'min'
          - 'lts'
```
(They are not identical)

---

What's min?

---

It tests the minimum compatible version.